### PR TITLE
Add a React Native sample

### PR DESCRIPTION
**Pull request by JoshMiers-FS**

### CODE REVIEW

**Comment by JoshMiers-FS** [comment]
Very simple RN app that is configured to use our new babel plugin.

**Comment by JoshMiers-FS** [comment]
https://app.clubhouse.io/fullstory/story/115713/sample-test-app-for-automated-tests-and-to-validate-react-native-works

File: samples/react-native-app/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+This project was bootstrapped with [Create React Native App](https://github.com/expo/create-react-native-app).
+
```
Comment (patrick-fs) [inline samples/react-native-app/README.md:2]: Could you provide brain-dead simple instructions on how to get going? Include how to install deps in the project directory, etc.

Comment (JoshMiers-FS) [reply samples/react-native-app/README.md:2]: yep, done

**Pull request merged**